tds/src/main/webapp/WEB-INF/templates/commonFragments.html

```diff
@@ -4,24 +4,14 @@
 <!--/* The indentation of the fragments below matches the indentation of the elements that they will replace in the
        target templates. Do not normalize it, or else the generated HTML won't be pretty any more. */-->
 
-<head th:fragment="head(title, scripts)">
-  <title th:replace="${title}">NetCDF Subset Service</title>
```

Comment:
Thymeleaf used to only be used for the netCDF Subset Service, but now thanks to you, we're rocking it out everywhere. However, it would still be good to make sure the NetCDF Subset Service pages have the proper title. Maybe it already does and this is from my lack of understanding the templating system. If so, my bad! If not, we should open an issue to make sure we get that working at some point.